### PR TITLE
Updated jvm and native Dockerfiles

### PR DESCRIPTION
**Pull request by jirisimek98**
Updated Dockerfiles since the outdated ones have issues running on OCP 4.11

### CODE REVIEW

File: quarkus-test-images/src/main/resources/Dockerfile.jvm [hunk 1]
```diff
@@ -7,48 +7,87 @@
...
 
-# Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
-ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
+
 # We make four distinct layers so if there are application changes the library layers can be re-used
-COPY --chown=1001 ${ARTIFACT_PARENT}/quarkus-app/lib/ /deployments/lib/
-COPY --chown=1001 ${ARTIFACT_PARENT}/quarkus-app/*.jar /deployments/
-COPY --chown=1001 ${ARTIFACT_PARENT}/quarkus-app/app/ /deployments/app/
-COPY --chown=1001 ${ARTIFACT_PARENT}/quarkus-app/quarkus/ /deployments/quarkus/
+COPY --chown=185 target/quarkus-app/lib/ /deployments/lib/
```
Comment (pjgg) [inline quarkus-test-images/src/main/resources/Dockerfile.jvm:84]: These docker files are not the final ones, are like "templates" and pre-processing by functions like this one: https://github.com/quarkus-qe/quarkus-test-framework/blob/main/quarkus-test-images/src/main/java/io/quarkus/test/utils/DockerUtils.java#L52

So could you replace the word `target` by `${ARTIFACT_PARENT}`

Line: 85, 86 and 87

File: quarkus-test-images/src/main/resources/Dockerfile.native [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+# docker run -i --rm -p 8080:8080 quarkus/code-with-quarkus
 #
 ###
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5
 WORKDIR /work/
 RUN chown 1001 /work \
     && chmod "g+rwX" /work \
     && chown 1001:root /work
-COPY --chown=1001:root ${ARTIFACT_PARENT}/*-runner /work/application
+COPY --chown=1001:root target/*-runner /work/application
```
Comment (pjgg) [inline quarkus-test-images/src/main/resources/Dockerfile.native:22]: Same "target" replacement is needed.

**Review by pjgg — CHANGES_REQUESTED** [review]
Thank you @jirisimek98 for your PR, just a few comments

**Review by fedinskiy — APPROVED** [review]
Looks good to me, but let's wait for CI

And could you bring back that newline at the end of Dockerfile.native? :-)

**Comment by fedinskiy** [comment]
run tests

**Comment by fedinskiy** [comment]
run tests

**Review by mjurc — APPROVED** [review]
CI failures are unrelated.

LGTM, thanks for the update!

**Pull request merged**